ansible/roles/stack-sunbird/tasks/assessment-service.yml

```diff
@@ -7,6 +7,10 @@
   become: yes
   template: src=assessment-service_application.conf dest=/home/deployer/config/assessment-service.conf mode=0644
 
+- name: Save logback configuration into a env file
```

Comment:
These are for swarm and it will not be required anymore @krgauraw

Reply:
As discussed, this will not be added for any other service.